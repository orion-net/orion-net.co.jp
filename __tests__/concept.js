import React from "react"
import renderer from "react-test-renderer"

import Main from "../src/pages/concept"

jest.mock("gatsby-plugin-image", () => {
  const StaticImageMock = () => <img>Mocked StaticImage component</img>

  return {
    StaticImage: StaticImageMock,
  }
})

describe("concept", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Main />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
