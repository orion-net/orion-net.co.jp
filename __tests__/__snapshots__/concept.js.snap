// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`concept renders correctly 1`] = `
Array [
  <div
    className="hamburger-menu"
  >
    <div
      className="bar-wrapper closed"
      onClick={[Function]}
    >
      <div
        className="bar bar0"
      />
      <div
        className="bar bar1"
      />
      <div
        className="bar bar2"
      />
    </div>
    <ul
      className="navbar closed"
      onClick={[Function]}
    >
      <li
        className=""
      >
        <a
          aria-current="page"
          className=""
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          style={Object {}}
        >
          トップページ
        </a>
      </li>
      <li
        className="current_page_item"
      >
        <a
          href="/concept"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          サロン紹介
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/menu"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          メニュー
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/custom"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          商品紹介
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/custom2"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          おすすめ
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/custom1"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          ご予約
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/access"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          アクセス
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/concept1"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          成人式・七五三
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/contact"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          お問い合わせ
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/privacy"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          プライバシーポリシー
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/custom3"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          成人式・スナップ集
        </a>
      </li>
    </ul>
  </div>,
  <div
    className="layout"
  >
    <div
      onClick={[Function]}
    >
      <div
        className="logo"
      >
        <a
          aria-current="page"
          className=""
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          style={Object {}}
        >
          <img
            alt="オリオン美容室のトップ"
            src="/img/top3.png"
          />
        </a>
      </div>
      <div
        className="article-wrapper"
      >
        <h1
          className="entry-title"
        >
          サロン紹介
        </h1>
        <h2>
          スタッフ情報
        </h2>
        <div
          className="staff"
        >
          <p>
            マネージャー：
          </p>
          <div
            className="manager-name"
          >
            武藤　夏代
          </div>
        </div>
        <div
          className="staffs"
        >
          <div
            className="staff"
          >
            <img>
              Mocked StaticImage component
            </img>
            <div
              className="staff-name"
            >
              武藤　大亮
            </div>
          </div>
          <div
            className="staff"
          >
            <img>
              Mocked StaticImage component
            </img>
            <div
              className="staff-name"
            >
              武藤　圭佑
            </div>
          </div>
        </div>
        <h2
          className="paragraph"
        >
          サロン内の写真
        </h2>
        <img>
          Mocked StaticImage component
        </img>
        <img>
          Mocked StaticImage component
        </img>
        <div
          className="discription"
        >
          <p>
            白を基調とした落ち着いた店内は、おくつろぎいただける空間です。
          </p>
          <p>
            お客さまの髪質とご要望に合わせて、スタイルをご提案します。
          </p>
        </div>
        <img>
          Mocked StaticImage component
        </img>
        <img>
          Mocked StaticImage component
        </img>
        <div
          className="discription"
        >
          <p>
            180度にたおれるイスがございますので、ゆったりと施術がお受けになれます。
          </p>
        </div>
      </div>
      <div
        className="sidebar"
      >
        <div>
          <a
            className="blog-button"
            href="https://ameblo.jp/teamorion/?frm_id=v.mypage-ameblo--myblog--blog"
          >
            ブログ
          </a>
        </div>
        <div>
          <a
            className="facebook-button"
            href="https://www.facebook.com/orionannex.orion"
          >
            Face Book
          </a>
        </div>
        <div
          className="sp-part-top sp-sns-button"
          id="sp-sns-button-1"
        >
          <ul>
            <li>
              <iframe
                allowtransparency="true"
                frameBorder="0"
                scrolling="no"
                src="https://www.facebook.com/plugins/like.php?href=http%3A%2F%2Forion-net.co.jp%2Fprivacy.html&layout=button_count&width=120&height=21&send=true&show_faces=true&share=false&action=like&colorscheme=light&font"
                style={
                  Object {
                    "border": "none",
                    "height": "21px",
                    "overflow": "hidden",
                    "width": "120px",
                  }
                }
              />
            </li>
            <li>
              <a
                className="twitter-share-button"
                data-count="horizontal"
                data-lang="ja"
                href="https://twitter.com/share"
              >
                Tweet
              </a>
              <script
                charSet="utf-8"
                src="https://platform.twitter.com/widgets.js"
                type="text/javascript"
              />
            </li>
            <li>
              <a
                href="https://line.me/R/msg/text/?%0D%0Ahttp%3A%2F%2Forion-net.co.jp%2Fprivacy.html"
              >
                <img
                  alt=""
                  height="20"
                  src="/img/linebutton_horizontal.png"
                  style={
                    Object {
                      "border": "none",
                    }
                  }
                  width="82"
                />
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>,
  <footer
    className="footer"
  >
    <div
      className="footer-link"
    >
      <a
        className="privacy-policy-link"
        href="/privacy"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        プライバシーポリシー
      </a>
    </div>
    <p
      className="copyright"
    >
      Copyright © オリオン美容室, All rights reserved.
    </p>
  </footer>,
]
`;
